common/perception_utils/include/perception_utils/object_classification.hpp

```diff
@@ -17,11 +17,27 @@
 
 #include "autoware_auto_perception_msgs/msg/object_classification.hpp"
 
+#include <vector>
+
 namespace perception_utils
 {
 using autoware_auto_perception_msgs::msg::ObjectClassification;
 
-bool isVehicle(const uint8_t object_classification)
+inline std::uint8_t getHighestProbLabel(
+  const std::vector<ObjectClassification> & object_classifications)
+{
+  std::uint8_t label = ObjectClassification::UNKNOWN;
+  float highest_prob = 0.0;
+  for (const auto & object_classification : object_classifications) {
+    if (highest_prob < object_classification.probability) {
+      highest_prob = object_classification.probability;
+      label = object_classification.label;
+    }
+  }
```

Comment:
To prevent bigger PR, I fix another PR.
So I add TODO at [bee5d5d](https://github.com/autowarefoundation/autoware.universe/pull/1954/commits/bee5d5d5175ffeefef0a7c5462a4771642434388).